packages/jsii-dotnet-runtime/src/Amazon.JSII.Runtime/Deputy/JsiiOptionalAttribute.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+namespace Amazon.JSII.Runtime.Deputy
+{
+    [AttributeUsage(AttributeTargets.Property)]
+    public class JsiiOptionalAttribute : Attribute
```

Comment:
I'd love some documentation on this... To explain exactly what it means (this one is obvious - but it's a good habit to build :D)

Reply:
Added a summary :-)